scripts/commands.js

```
@@ -169,8 +169,91 @@ async function main() {
+        await checkAccounts();
+      } else if (checkSubCommand === 'all') {
+        await runFullCheck();
+      } else {
+        console.log('Running full health check...');
+        await runFullCheck();
+        console.log('Check options: deployment, config, tables, accounts, all')
+      }
+      break;
+
+    case 'test':
+      const testSubCommand = process.argv[3];
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)